buy-resell/products/middleware.md

```diff
@@ -54,4 +54,73 @@ import HostedMiddlewareDE from '../../_markets/de/buy-resell/products/_hosted-mi
     <HostedMiddlewareDE />
   </TabItem>
 
+</Tabs> -->
```

Comment:
I am not sure that I understand why we link the sections to check and edit outlets in the products middleware section ? 
I would rather expect the https://docs.fiskaltrust.cloud/docs/posdealers/getting-started/operator-onboarding/master-data#work-steps-to-check-master-data to be extended, so it lists the digital receipt as one of the reasons why checking the outlet data is important ?

Reply:
digital receipt is a product, positioning in the Table of contents was decided by digital receipt team